examples/ios/swift/Migration/README.md

```diff
@@ -14,8 +14,14 @@ You can build and run the project as is. Migrations from all prior version to th
    - v3 -> v5
    - v4 -> v5
 
-If you want to compare older versions among each other (i.e. v2 -> v3) you can do so by uncommenting the
-target version in `Migration.xcconfig`. 
+If you want to compare older versions among each other (i.e. v2 -> v3) you can do so by editing the `Preprocessor Macros` in the build settings.
```

Comment:
Should add that they can just edit the header files themselves (which would probably be more idiomatic).

Reply:
@jsflax  Not sure I understand it the way you mean. Editing the header files will not change the realm files already generated.
Can you make an example of what you're aiming for?

Reply:
For example, in Example_v2.h:
```
#ifndef SCHEMA_VERSION_2
#define SCHEMA_VERSION_2 0
#endif
```
can be changed to
```
#ifndef SCHEMA_VERSION_2
#define SCHEMA_VERSION_2 1
#endif
```
to enable SCHEMA_VERSION_2. However, your change to `Example_v5.h` was incorrect. 

```
if SCHEMA_VERSION_5
```
should be
```
#if SCHEMA_VERSION_5 && !SCHEMA_VERSION_4 && !SCHEMA_VERSION_3 ...
```
When I stated `This will need to be updated to reflect the objc approach to macros I mentioned above`, it meant to account for truthiness, not just definition.

Thís will allow users to edit the header files directly (which is more idiomatic) as opposed to having to go through Xcode's user defined variables.